inmemory_repository.go

```diff
@@ -73,3 +73,7 @@ func (r *InMemoryRepository) EventsSince(timestamp time.Time, limit int) ([]Even
 
 	return events, nil
 }
+
+func (r *InMemoryRepository[T]) Update(objectID string, object T, nbRetries int, updater func(T) (T, error)) (T, error) {
```

Comment:
```suggestion
func (r *InMemoryRepository[T]) Update(objectID string, nbRetries int, updater func(T) (T, error)) (T, error) {
```

Si plus besoin de passer au Load.